packages/js/src/ui/context/AppearanceContext.tsx

```diff
@@ -12,6 +12,7 @@ export type ElementStyles = string | CSSProperties;
 export type Elements = {
   button?: ElementStyles;
   root?: ElementStyles;
+  bell?: ElementStyles;
```

Comment:
This should probably be `bellContainer` and the `bell` will be for the icon itself.